tests/distances/mcconnaughey.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pytest
+from scipy.sparse import csr_array
+
+from skfp.distances import mcconnaughey_binary_distance, mcconnaughey_binary_similarity
+from tests.distances.utils import assert_similarity_and_distance_values
+
+
+def _get_values() -> list[tuple[list[int], list[int], str, float, bool]]:
+    return [
+        ([1, 0, 0], [0, 1, 1], "==", 0.0, True),
+        ([1, 0, 0], [0, 0, 0], "==", 0.0, True),
```

Comment:
Are those values correct? If it ranges from -1 to 1, we should have tests for -1, 0 and 1 (extremes), as well as some for the normalized variant